assets/css/campfire-games.css

```diff
@@ -0,0 +1,42 @@
+h2 {
+  margin-bottom: 20px;
+}
+
+.container {
+  padding-top: 30px;
+}
+
+#teams-container {
+  background-color: rgb(227,219,203);
+}
+
+.marshie {
+  height: 200px;
+  width: 200px;
+  margin-bottom: 5px;
+}
+
+.row {
+  padding-top: 60px
+}
```

Comment:
This style is causing some spacing issues on mobile--maybe you want to decrease this value at the medium breakpoint (`768px`)?
![image](https://user-images.githubusercontent.com/6632788/109360880-f3a28600-7855-11eb-9e59-bfb0065ddfa6.png)